Automattic/wp-calypso
# Rename /plans page images to avoid getting blocked by ad blockers

#### Changes proposed in this Pull Request

This PR changes the names of images that are being blocked by ad blockers ([such as AdBlock](https://getadblock.com/)).
The [EasyList filter](https://easylist.to/easylist/easylist.txt) used by the ad blocker contains patterns which hide `ads-removed.svg`, `google-adwords.svg`, and `advertising-removed.svg` images.

The issue fixed by reversing the words for each image.

More information here 474-gh-martech

#### Testing instructions

TL;DR With AdBlock extension, Go through the `Thank you` and `My Plan` pages for each paid plan. The `Advertising removed` and `Google Ads credit` should always have a corresponding illustration.

1. Install AdBlock. It should have EasyList enable by default.
2. Upgrade to Personal
   - Thank you and My plan pages should have the Advertising Removed image (https://d.pr/i/692JQd)
3. Upgrade to Premium
   - Thank you and My Plan pages should have the Advertising Removed and Google Ads credit images (https://d.pr/i/T1ZX9I)
4. Upgrade to Business
   - My Plan should have Advertising Removed and Google Ads credit images 
5. Upgrade to eCommerce
   - My Plan should have Advertising Removed and Google Ads credit images
